library/content.js

```diff
@@ -1,53 +1,81 @@
 import db from "./db";
+const { validate } = require("uuid");
+const log = require("loglevel");
 
-export async function getType(contentId) {
-  const track = await db
+export const getParentContentTypeAndId = async (contentId) => {
+  const album = await db
     .knex("track")
-    .select("id")
+    .select("album_id")
     .where("track.id", "=", contentId)
     .then((data) => {
-      return data.length > 0;
+      return data[0].album_id;
```

Comment:
will this error if no `album` is found?

data[0] may be undefined, so something like error reading `album_id` of undefined
```suggestion
      return data[0]?.album_id;
```

Reply:
Wrote a test for each of these and fixed